net/net-tools/Makefile

```diff
@@ -53,6 +53,22 @@ define Package/net-tools-route/description
 	functionality of the net-tools variant (e.g. AF_X25).
 endef
 
+define Package/net-tools-netstat
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=net-tools - netstat utility
+  URL:=http://net-tools.sourceforge.net/
```

Comment:
This could be changed to https or even the page itself says that the project home is here: https://sourceforge.net/projects/net-tools/